mlflow/store/file_store.py

```diff
@@ -585,6 +585,15 @@ def set_tag(self, run_id, tag):
         # Don't add trailing newline
         write_to(tag_path, self._writeable_value(tag.value))
 
+    def delete_tag(self, run_id, key):
+        _validate_run_id(run_id)
+        run = self.get_run(run_id)
+        check_run_is_active(run.info)
+        if key not in run.data.tags.keys():
+            raise MlflowException("No tag with name: {} in run with id {}".format(key, run_id))
```

Comment:
This exception should have `error_code = mlflow.protos.databricks_pb2.RESOURCE_DOES_NOT_EXIST`